### DISCUSSION TITLE
Assignment 1 Due Date

### DISCUSSION THREAD

**Discussion by NicolasFranklin** [chulminy/CIVE497-CIVE700#110] [General]
![image](https://user-images.githubusercontent.com/40611754/104769589-87544300-573d-11eb-9d82-e81150fba166.png)

I noticed on the course home assignment 1 is due on Friday the 22nd, but on the actual task1 page it says the 20th.

Can you clarify which date is correct? Thanks.

**Comment by chulminy** [comment]
You are correct! Fixed!
